src/publish.js

```diff
@@ -29,7 +29,7 @@ const redirects = require('./fastly/redirects');
  * @param {object} iconfig the IndexConfig from helix-shared
  * @param {string} algoliaappid Algolia App ID (not secret)
  */
-async function publish(configuration, service, token, version, vclOverrides = {}, dispatchVersion = 'v3', log = console, iconfig, algoliaappid) {
+async function publish(configuration, service, token, version, vclOverrides = {}, dispatchVersion = 'v3', log = console, iconfig, algoliaappid, wskAuth, wskHost, wskNamespace) {
```

Comment:
this gets longer and longer....maybe switch to a different style. eg:
- pass a `options` object
- use a builder pattern

(but this can be addresses in a different PR)